src/inference/dev_api/cpp_interfaces/interface/ie_internal_plugin_config.hpp

```diff
@@ -65,6 +65,12 @@ DECLARE_CONFIG_KEY(FORCE_DISABLE_CACHE);
  */
 DECLARE_CONFIG_KEY(CONFIG_DEVICE_ID);
 
+/**
+ * @brief Defines if Softmax can be tokenized in Snippets
+ * @ingroup ie_dev_api_plugin_api
+ */
+DECLARE_CONFIG_KEY(SNIPPETS_SOFTMAX_ENABLE);
```

Comment:
Do we need this key exclusively to run Softmax tests? If so, it's worth to add a todo like: remove when Softmax tokenization is fully supported in snippets + Softmax support ticket ID